@navikt/core/css/darkside/accordion.darkside.css

```diff
@@ -1,198 +1,168 @@
+.navds-accordion__item[data-expanded="false"]:has(.navds-accordion__header:where(:hover, :focus-visible))
+  + .navds-accordion__item
+  > .navds-accordion__header::before {
+  display: none;
 }
 
-/*************************
- * Icon *
- *************************/
+/* ------------------------- Accordion chevron icon ------------------------- */
 .navds-accordion__icon-wrapper {
   display: grid;
   place-content: center;
-  border-radius: var(--a-border-radius-medium);
-  height: var(--a-spacing-6);
-  width: var(--a-spacing-6);
+  border-radius: var(--ax-border-radius-large);
+  width: 22px;
+  height: 22px;
   align-self: center;
-}
-
-.navds-accordion__header:hover > .navds-accordion__icon-wrapper {
-  background-color: var(--ac-accordion-header-icon-bg-hover, var(--a-surface-neutral-subtle-hover));
+  background-color: var(--ax-bg-accent-moderateA);
+  color: var(--ax-text-accent-strong);
 }
 
 .navds-accordion__header-chevron {
-  border-radius: var(--a-border-radius-medium);
-  font-size: 1.5rem;
-  height: 1.75rem;
+  height: inherit;
+  width: inherit;
   flex-shrink: 0;
-  transition: transform 150ms ease-in-out;
-  align-self: center;
-}
-
-:where(.navds-accordion__header):hover > :where(.navds-accordion__icon-wrapper) > :where(.navds-accordion__header-chevron) {
-  transform: translateY(1px);
-}
-
-.navds-accordion__item--open
-  > :where(.navds-accordion__header)
-  > :where(.navds-accordion__icon-wrapper)
-  > :where(.navds-accordion__header-chevron) {
-  transform: translateY(0) rotate(-180deg);
+  transition: transform 250ms cubic-bezier(0.2, 0, 0, 1);
 }
 
-.navds-accordion__item--open
-  > :where(.navds-accordion__header):hover
-  > :where(.navds-accordion__icon-wrapper)
-  > :where(.navds-accordion__header-chevron) {
-  transform: translateY(-1px) rotate(-180deg);
+.navds-accordion__item--open > .navds-accordion__header .navds-accordion__header-chevron {
+  transform: rotateX(-180deg);
 }
 
-/*************************
- * Variant/Default *
- *************************/
-.navds-accordion__item--open {
-  background-color: var(--ac-accordion-item-bg-open, var(--a-transparent));
-}
-
-/*************************
- * Variant/Neutral *
- *************************/
-.navds-accordion__item--open.navds-accordion__item--neutral {
-  background-color: var(--ac-accordion-neutral-item-bg-open, var(--a-surface-neutral-subtle));
-}
-
-/*************************
- * Content *
- *************************/
-
+/* ---------------------------- Accordion content --------------------------- */
 .navds-accordion__content {
-  padding: var(--a-spacing-1) var(--a-spacing-3) var(--a-spacing-5);
-  animation: fadeAccordionContent 250ms ease;
-}
+  --__acx-accordion-content-line-width: 2px;
 
-.navds-accordion--indent > :where(.navds-accordion__item) > :where(.navds-accordion__content) {
-  padding: var(--a-spacing-1) var(--a-spacing-3) var(--a-spacing-5) var(--a-spacing-11);
-}
-
-.navds-accordion__content--closed {
+  position: relative;
```

Comment:
This will probably reintroduce [this](https://nav-it.slack.com/archives/C7NE7A8UF/p1727161300911749) 🤔

Reply:
Yep thought that might happend 😭 The required design needs some changes to the React-code for accordion if not using relative, so tried avoiding that. 
Need to think about some solutions then
- Feature-flagging use of new syntax with a context we create could work
- Or just "hacking" another item into the AccordionContent and setting it to default display none so the old system ignores it

Reply:
Not sure I understand, do you mean that the changes required to remove position:relative would be breaking?

Maybe we should create a separate "FilterGroup" component... 🤔

Reply:
We discussed just exposing the "Collapsible"-component used in FormProgress so that those that need more custom accordions or collapsibles can use that